src/sentry/static/sentry/app/components/alert.tsx

```diff
@@ -67,7 +67,7 @@ const alertStyles = ({
   thinner,
 }: Props & {theme: any}) => css`
   display: flex;
-  margin: 0 0 ${space(3)};
+  margin: 0 0 ${space(thinner ? 0 : 3)};
```

Comment:
I wouldn't remove margins here, I would remove it where you are using the Alert, otherwise we have two different behaviors for the margin. When I see `thinner`, I probably think about its own size, not how far away it is from other elements.

Reply:
Thanks, done!